provision/provision-site.sh

```diff
@@ -102,6 +133,10 @@ function vvv_provision_site_nginx_config() {
   if [ 'php' != "${NGINX_UPSTREAM}" ] && [ ! -f "/etc/nginx/upstreams/${NGINX_UPSTREAM}.conf" ]; then
     vvv_error " * Upstream value '${NGINX_UPSTREAM}' doesn't match a valid upstream. Defaulting to 'php'.${CRESET}"
     NGINX_UPSTREAM='php'
+    DEFAULTPHP=$(vvv_get_site_config_value 'php' "")
+    if [ '' != "${DEFAULTPHP}" ]; then
+      NGINX_UPSTREAM=$DEFAULTPHP
+    fi
   fi
   sed -i "s#{upstream}#${NGINX_UPSTREAM}#"  "${TMPFILE}"
```

Comment:
I don't think this will work, if no `nginx_upstream` is provided then its value will default to `php` and none of this code will run. This code is checking if the nginx upstream the user has specified in the config actually exists, and resets it to `php` if it does not